ui/app/components/dashboard/quick-actions-card.js

```diff
@@ -50,7 +51,10 @@ export default class DashboardQuickActionsCard extends Component {
           buttonText: 'Read secrets',
           // check kv version to figure out which model to use
           model: this.selectedEngine.version === 2 ? 'secret-v2' : 'secret',
-          route: 'vault.cluster.secrets.backend.show',
+          route:
+            this.selectedEngine.version === 1
+              ? 'vault.cluster.secrets.backend.show'
+              : 'vault.cluster.secrets.backend.kv.secret.details',
```

Comment:
Switching this to match the ternary statement above might read a little easier
```suggestion
            this.selectedEngine.version === 2
              ?  'vault.cluster.secrets.backend.kv.secret.details'
              : 'vault.cluster.secrets.backend.show',
```